<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificaUtente</class>
 <widget class="QDialog" name="ModificaUtente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifica Utente</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(159, 197, 248)</string>
  </property>
  <widget class="QLineEdit" name="tbUsername">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>37</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbPassword">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>95</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
    </font>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btConferma">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(132, 224, 142)</string>
   </property>
   <property name="text">
    <string>Conferma</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
